Arrowgene.Ddon.GameServer/Handler/QuestSetPriorityQuestHandler.cs

```diff
@@ -32,11 +31,27 @@ public override S2CQuestSetPriorityQuestRes Handle(GameClient client, C2SQuestSe
                     Server.Database.DeletePriorityQuest(client.Character.CommonId, questScheduleId);
                     continue;
                 }
-
                 var quest = QuestManager.GetQuestByScheduleId(questScheduleId);
+                if (quest == null)
+                {
+                    Logger.Error(client, $"No quest object exists for ${questScheduleId}");
+                    continue;
+                }
+
                 var questStateManager = QuestManager.GetQuestStateManager(client, quest);
+                if (questStateManager == null)
+                {
+                    Logger.Error(client, $"Unable to fetch the quest state manager for ${questScheduleId}");
+                    continue;
+                }
+
                 var questState = questStateManager.GetQuestState(questScheduleId);
```

Comment:
Should the function that gets the quest state encapsulate some of these things? Looking at the fixes, it looks like we do the same steps in multiple handlers.